### PR TITLE
Adds a config parameter to set the m-line direction.

### PR DESCRIPTION
**Pull request by gpolitis**

### CODE REVIEW

**Comment by saghul** [comment]
@gpolitis Can you elaborate about why this is needed? I can see what it does, but I don't understand why :-)

**Comment by gpolitis** [comment]
yeah, some context is missing here ;) so I wanted to be able to launch chrome instances (from the command line) that are send only for testing purposes (so I needed a config option that can be set from the URL).

**Comment by saghul** [comment]
IMHO this should use a private a more descriptive name, such as "_sdpAnswerDirection" + some documentation in config.js.

**Comment by gpolitis** [comment]
@saghul  I can document the parameter in config.js; however, I'm not sure I understand what you would like me to change. Would you like me to rename '_setAnswerDirection` to `_sdpAnswerDirection`?

**Comment by saghul** [comment]
No, I mean the option name itself. "direction" is not very descriptive... :-)

File: modules/RTC/TraceablePeerConnection.js [hunk 1]
```diff
@@ -945,6 +967,11 @@ TraceablePeerConnection.prototype.createAnswer
                 _fixAnswerRFC4145Setup(remoteDescription, localDescription);
                 answer.sdp = localDescription.raw;
 
+                if (answer && this.options._sdpAnswerDirection) {
```
Comment (saghul) [inline modules/RTC/TraceablePeerConnection.js]: you don't need to check for `answer` it will be there or the live above would have thrown

**Comment by saghul** [comment]
Also, you'll need to rebase, now that P2P landed.

**Comment by gpolitis** [comment]
As explained in https://github.com/jitsi/jicofo/pull/181 this breaks stuff: If a participant joins video muted, then the m-line direction is set to recvonly and the colibri channel in the bridge gets configured as sendonly and it's not getting updated.

**Pull request closed without merging**